CenterForOpenScience/osf.io
# [Hotfix][PLAT-973] fix 'qatest' tags so they freeze Crossref updates.

## Purpose

Putting a 'qatest' tag on a preprint is supposed to make that preprint stop updating to Crossref, but oops it doesn't. This PR fixes that.

## Changes

- Adds check to celery task to prevent updating on field changes.

## QA Notes

Add a 'qatest' tag to a preprint while making a preprint, the preprint should not send data to Crossref. If the tag is added while the preprint already made it should freeze updates, when the tag is removed updates should resume.

## Documentation

Not a user facing change.

## Side Effects

We might want to check out the Crossref processing queue now and then I noticed on staging we seem to have 2,000 changes queued up, it would be nice to know we're not also getting backed up on prod. Probably we aren't but I'm curious.

## Ticket

https://openscience.atlassian.net/projects/PLAT/issues/PLAT-973
